src/actions/index.js

```diff
@@ -0,0 +1,14 @@
+//#region > Imports
+//> Core
+import { ferry } from "./core";
+//#endregion
+
+//#region > Export
+//> Core
+export { ferry };
+//#endregion
+
+/**
+ * SPDX-License-Identifier: (EUPL-1.2)
+ * Copyright © 2019-2020 Simon Prast
```

Comment:
Remove the year number for consitency

Reply:
I will add the year to the missing tags for consistency.